src/hotspot/share/opto/output.cpp

```diff
@@ -1871,19 +1884,15 @@ void PhaseOutput::fill_buffer(CodeBuffer* cb, uint* blk_starts) {
       }
       if (C->method() != NULL) {
         tty->print_cr("----------------------- MetaData before Compile_id = %d ------------------------", C->compile_id());
-        C->method()->print_metadata();
+        tty->print("%s", method_metadata_str.as_string());
```

Comment:
is print_raw() better than  print here?  at least you don't need to handle vararg.

Reply:
Agree. I changed it